tsconfig.json

```diff
@@ -1,22 +1,27 @@
 {
   "compilerOptions": {
-    "baseUrl": "src",
-    "rootDir": "./",
     "outDir": "build",
-    "target": "esnext",
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "declaration": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
     "module": "commonjs",
     "moduleResolution": "node",
-    "sourceMap": true,
-    "removeComments": true,
-    "strict": true,
     "resolveJsonModule": true,
-    "esModuleInterop": true,
-    "allowJs": true,
-    "declaration": true,
+    "isolatedModules": true,
```

Comment:
Why did you enable `isolatedModules`?

Reply:
It perform additional checks to ensure that separate compilation such as with babel would be safe. I assume a common way of digesting xpring-common-js ,the way Xpring Wallet will be using it, will be through babel.

Reply:
Removed this for now

There is some additional information on babel about this if you are interested:
https://babeljs.io/docs/en/babel-plugin-transform-typescript#typescript-compiler-options
https://babeljs.io/docs/en/babel-plugin-transform-typescript